modules/mono/editor/GodotTools/GodotTools/Internals/GodotSharpDirs.cs

```diff
@@ -66,6 +66,13 @@ static string DetermineProjectName()
                 ProjectSettings.SetSetting("dotnet/project/assembly_name", _projectAssemblyName);
             }
 
+            _projectSolutionName = (string)ProjectSettings.GetSetting("dotnet/project/solution_name");
```

Comment:
```suggestion
            string projectSolutionName = (string)ProjectSettings.GetSetting("dotnet/project/solution_name");
```
And adjust all the other `_projectSolutionName`